### ISSUE TITLE
Support for 20.04 in Ubuntu Haskell PPA

### ISSUE THREAD

**Issue by AlistairB** [haskell-CI/haskell-ci#382]
Can the newly released Ubuntu version be supported in https://launchpad.net/~hvr/+archive/ubuntu/ghc ?

I have tested installing `ghc-8.10.1` on 20.04 using the eoan releases and it seems to work fine.

Happy to help if I can.

**Comment by phadej** [comment]
Yes, it can (and will) be supported. No ETA promises though.

**Comment by hvr** [comment]
I plan to take care of this over the upcoming weekend!

**Comment by hvr** [comment]
Minor update: I've been running into issues w/ some versions segfaulting with some packages which I still need to investigate

Current status: The PPA already features working GHC 7.0.1 / 7.0.4 / 7.2.2 / 7.10.3 / 8.0.2 / 8.2.2 / 8.4.4 / 8.6.5 / 8.10.1 packages (please test them in case I missed problems)

Do not use the GHC 7.4.2 Focal package from the PPA -- it doesn't work properly; GHC 7.6.3 and 7.8.4 are missing because they suffer from the same segfaulting issue

**Comment by phadej** [comment]
is 8.8.3 omitted on purpose?

**Comment by AlistairB** [comment]
@hvr Thanks! I am using it now. I am mostly using it with 8.8.3. No issues so far.

**Comment by hvr** [comment]
> is 8.8.3 omitted on purpose?

no, just skipped it accidentally in my enumeration; GHC 8.8.3 is fine too, and it was uploaded into the PPA together with the other "good" versions.

**Comment by 23Skidoo** [comment]
I guess that this ticket can be closed?

**Comment by yrogge** [comment]
I have tested focal versions 8.10.x on Ubuntu 20.10 (groovy) with success.
Please upgrade PPA with a groovy version for recent releases.
Thanks

**Comment by AlistairB** [comment]
I'm assuming this is all good now

**Comment by hvr** [comment]
I'm afraid the issues isn't resolved yet; there's still the 3 major versions GHC 7.4/7.6/7.8 which don't work yet (Oleg provided me a lead about what might be the cause which I still need to pursue). 

In fact, this is currently still preventing me from upgrading my Ubuntu 18.04 based development workstations as I need those versions to work properly.
